platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
@werwty I can't agree with you on that. What you propose is half solution because then on the end task you would end up with `task(*args, kwargs={kwargs: {...}})`"and also you would loose this way ability to pass all additional arguments to apply_async.